sli/slistartup.cc

```diff
@@ -209,7 +203,7 @@ SLIStartup::checkenvpath( std::string const& envvar, SLIInterpreter* i, std::str
 SLIStartup::SLIStartup( int argc, char** argv )
   : startupfilename( "sli-init.sli" )
   , slilibpath( "/sli" )
-  , slihomepath( NEST_INSTALL_PREFIX "/" NEST_INSTALL_DATADIR )
+  , slilibdir( NEST_INSTALL_PREFIX "/" NEST_INSTALL_DATADIR )
```

Comment:
Why is this the "data" directory and not a "lib" directory?

Reply:
`NEST_INSTALL_DATADIR` contains all non-binary data files for NEST and the SLI standard library is considered as such data.